app/mailers/jobseekers/message_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class Jobseekers::MessageMailer < Jobseekers::BaseMailer
+  def message_received(message)
+    @message = message
+    @jobseeker = message.conversation.job_application.jobseeker
+    @job_application = message.conversation.job_application
+    @vacancy = @job_application.vacancy
+    @organisation = @vacancy.organisation
+
+    @subject = if @job_application.status == "unsuccessful"
```

Comment:
Non-blocking suggestion:
```suggestion
    @subject = if @job_application.unsuccessful?
```